committee-code-of-conduct/election.md

```diff
@@ -0,0 +1,80 @@
+# Kubernetes Code of Conduct Committee Elections
+
+This document outlines the process, for Code of Conduct Committee (CoCC) elections.
+
+### Length of Terms
+
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed for 2 year terms and the other 2 members will be appointed for a 1 year term.
+* In all subsequent elections the term will be for 2 years.
+* In alternating years either 2 or 3 seats will be up for election as terms expire.
+
+### Eligibility for voting
+
+* Voting will be done by members of the steering committee.
+
+### Eligibility for candidacy
+
+* Candidate commits to putting the interests of the community above the interests of their employer for all Code of Conduct Committee activities.
+* Is generally a responsible human
+* Does not have to be part of the Kubernetes or CNCF community
```

Comment:
I'd like to understand this one a little better.  We expect this of our leadership roles in all other community groups, why not here?

Reply:
The general idea is we can leverage expertise in the area of code of conduct from other communities by not explicitly requiring someone to be part of the k8s and cncf community. For example, it'd be great if someone from the Mozilla community which has lots of experience and expertise developing CoC processes came along to be part of our CoCC. The folks that are deeply involved in Kubernetes/CNCF may not be the same people who have a background in CoC issues and removing the requirement that these people have to come from our specific community will open up the possibility to having a more diverse and well versed CoCC.

Reply:
Sourcing from outside of our echo chamber makes sense, and I agree outside perspective is a healthy check.  So, for eligibility for candidacy, makes sense, LGTM

Once elected, they need to be members for all of our tooling (and ideally awareness of how this community operates), which I think we all know, I just couldn't find it spelled out anywhere.  Can address in a followup.

Reply:
Also, just so I'm explicitly clear: I read this as saying steering committee membership does not prevent code of conduct committee membership (or vice-versa).  The membership of the two committees is mutually exclusive today, but nothing in the charters or election docs prevents that from changing.  Correct?

Reply:
There is nothing that I can find which says they are mutually exclusive today. I'm not sure why they need be.